### PR TITLE
Implement revert button for revision (change culprit)

### PR DESCRIPTION
**Pull request by florinbilt**
How to Change the Revision of an Alert

Prerequisites:
You must be authenticated to perform this action.
The "Reset Revision" button is only visible if the alert's initial revision has been modified.

---

Steps

1. Open Browser Developer Tools
Open your browser's console (usually by pressing F12 or Ctrl+Shift+I).
Navigate to the Network tab.

2. Use the "Take" Button
Click the Take button to generate a request to the backend.

3. Copy the Request as cURL
Locate the request generated by the Take button in the Network tab.
Right-click the request and select Copy > Copy all as cURL.

4. Modify the cURL Request
Paste the copied cURL request into a text editor.
Find and remove the assignee_username key from the request.
Replace it with:
"revision": "hash_revision"

(Replace hash_revision with the desired revision hash.)

5. Execute the Modified Request
Open a terminal on your system.
Paste the modified cURL command into the terminal and press Enter.

---

Executing this command will update the alert's revision to the specified hash.

### CODE REVIEW

File: ui/perfherder/alerts/AlertHeader.jsx [hunk 1]
```diff
@@ -150,6 +155,19 @@ const AlertHeader = ({
...
+          alertSummary.original_revision !== alertSummary.revision && (
+            <Col className="p-0" xs="auto">
+              <Button
+                className="ml-1"
+                size="xs"
+                color="warning"
+                onClick={handleRevertRevision}
+              >
+                Revert
+              </Button>
```
[the remark this answers is not in the document]
Comment (florinbilt) [reply ui/perfherder/alerts/AlertHeader.jsx]: I moved the button before  Revision hash (Revert button Revision hash Author / Take button)

File: ui/perfherder/alerts/AlertHeader.jsx [hunk 2]
```diff
@@ -150,6 +155,19 @@ const AlertHeader = ({
             user={user}
           />
         </Col>
+        {user.isStaff &&
+          alertSummary.original_revision !== alertSummary.revision && (
+            <Col className="p-0" xs="auto">
+              <Button
+                className="ml-1"
+                size="xs"
+                color="warning"
```
[the remark this answers is not in the document]
Comment (florinbilt) [reply ui/perfherder/alerts/AlertHeader.jsx]: I left the default color. Now it looks like the "Take" button

**Comment by florinbilt** [comment]
Screenshot alertHeader

File: ui/perfherder/alerts/AlertHeader.jsx [hunk 3]
```diff
@@ -74,6 +79,14 @@ const AlertHeader = ({
             </a>
           </Row>
         </Col>
+        {user.isStaff &&
+          alertSummary.original_revision !== alertSummary.revision && (
+            <Col className="p-0" xs="auto">
+              <Button className="ml-1" size="xs" onClick={handleRevertRevision}>
+                Reset hash
```
Comment (esanuandra) [inline ui/perfherder/alerts/AlertHeader.jsx:86]: This needs to be changed into `Reset Revision` so it's consistent with the `Edit Revision` button that will come.

Comment (florinbilt) [reply ui/perfherder/alerts/AlertHeader.jsx]: Done

**Review by esanuandra — APPROVED** [review]

**Pull request merged**